Spring-2019/linked_lists/rotate_list/rotate_list.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+using namespace std;
+
+template <typename T>
```

Comment:
I feel like generics makes the solution a bit more convoluted than it needs to be. Maybe just assume that it's a linked list of ints?

Reply:
fair enough~

Reply:
Fixed solution method and driver files in cb831b7e3c703adf1b6403eaa6002e61f9722caa